ibc/core/events.go

```diff
@@ -0,0 +1,41 @@
+package core
+
```

Comment:
Could we add comments to the top of these files explaining that they were copy/pasted from an outdated version of IBC and are legacy. Prob useful to mark for future devs that these are not stable files to build out more testing functionality